demo_nodes_cpp/src/parameters/ros2param.cpp

```diff
@@ -139,6 +139,12 @@ int main(int argc, char ** argv)
       fprintf(stderr, "Failed to set parameter\n");
       return 1;
     }
+    auto results = set_parameters_result.get();
+    for (auto result : results) {
+      if (result.successful != true) {
```

Comment:
The rest of the file is using `!=`, should I change those as well, just this, or no change to keep it consistent?